.github/workflows/update-mbedtls.yml

```diff
@@ -37,6 +37,11 @@ jobs:
       pull-requests: write
     steps:
       - uses: actions/checkout@v4
+      - name: Compile mbedtls
```

Comment:
Hmm. But then this step makes `compile-mbedtls` job redundant. Can we somehow reuse the outcome of the `compile-mbedtls` job?

Reply:
I don't think its redundant since the different platforms require unique configurations for running, maybe pass the .kt from the linux compiler as a separate artifact?

Reply:
Yes, let's better try this way.